service/domain/agreements.ts

```diff
@@ -1,73 +1,27 @@
 export class AgreementListItem {
   id: string | null;
   name: string;
-  description?: string;
-  creationTime?: Date;
-
-  constructor(
-    id: string | null,
-    name: string,
-    description?: string,
-    creationTime?: Date
-  ) {
-    this.id = id;
-    this.name = name;
-    this.description = description;
-    this.creationTime = creationTime;
-  }
+  description: string | null;
+  creationTime: Date | null;
 }
 
 export class AgreementVersion {
   id: string;
-  agreementId?: string;
+  agreementId?: string | null;
   current: boolean;
   draft: boolean;
   texts?: AgreementText[];
   creationTime: Date;
-
-  constructor(
```

Comment:
Any reason to keep these (`AgreementVersion`, `AgreementText`, `Agreement`) as classes now that they don't have a constructor?

Reply:
I'm not sure.  There are other method-less classes in the codebase.